css/dashboard.css

```diff
@@ -0,0 +1,51 @@
+.dashboard {
+  flex: 0 0 800px;
+  align-content: center;
+  flex-flow: row wrap;
+  /* The top margin places the dashboard below the tabstrip, the bottom margin
+  visually centers the tiles vertically. We need the bottom margin because we're
+  using flexbox to align the tiles, but the tiles themselves are not visually
+  even, since they have extra space below for the title. */
+  margin: 100px auto 30px;
+  width: 800px;
+}
```

Comment:
These styles don't do what is intended... they center an uneven number of tiles (should be left-justified) and also align tiles to the top of the screen space, rather than centering them in space below tab strip.

Will update to be responsive.